### ISSUE TITLE
tag invokement

### ISSUE THREAD

**Issue by basz** [FortAwesome/ember-fontawesome#87]
```
{{fa-icon "igloo"}}
```
works but;

```
<i class="fas fa-igloo"></i>
```

does not.

Do I need to do something special?

environment.js contains;

```
fontawesome: {
      warnIfNoIconsIncluded: true,
      defaultPrefix: 'fas',
      icons: {
        'free-solid-svg-icons': 'all'
      }
    },
```

**Comment by jrjohnson** [comment]
For the most part you should stick with the `{{fa-icon}}` component, but if you need to replace specific `<i>` tags there is a helper for that. Using it in a component looks like:

```
import Component from '@ember/component';
import { dom } from '@fortawesome/fontawesome-svg-core';
import { next } from '@ember/runloop';

export default Component.extend({
  /**
   * Convert `<i>` into SVG icons
   * Uses: https://fontawesome.com/how-to-use/with-the-api/methods/dom-i2svg
   */
  didRender() {
    next(() => {
      dom.i2svg({node: this.element});
    });
  },
});
```

**Comment by jrjohnson** [comment]
I believe this is fixed by #88, but please re-open if I'm wrong.
